bazel/python_test.bzl

```diff
@@ -34,7 +34,7 @@ from bazel.python_utils.coverage_decorator import coverage_decorator
 
 if __name__ == "__main__":
     with coverage_decorator():
-        args =  ["-ra", "-vv"]  + %s + sys.argv[1:] + %s
```

Comment:
Why is this needed?

Reply:
This makes pytest show the output in real time instead of only after failures.